demo/_data.yml

```diff
@@ -9,3 +9,9 @@ metas:
   description: This is an example of a Lume blog theme
   twitter: "@misteroom"
   lang: "=lang"
+
+mergedKeys:
+  static_text: object
+
+static_text:
+  toc_name: Table of Contents
```

Comment:
- I think all texts should be defined by default in a _data file, so it's easier to know which texts are translatable.

- Maybe these copies should be saved into `src/_data/static_text.yml`, so they are independent of the rest of the data, and are accessible from `static_text.toc_name`, `static_text.next_post` etc. This allows to override these texts in your local installation without affecting to the other data files and viceversa.

- The variable name `static_text` maybe is too long? I'd go with something shorter, like `i18n` or `texts`.
- Instead of `toc_name`, I think `toc_title` is better.